open-api-spec.yaml

```diff
@@ -32,6 +32,8 @@ tags:
     description: outgoing payment operations
   - name: quote
     description: quote operations
+  - name: interledger
```

Comment:
I almost think `interledger` is too broad. Why not `stream` or `stream-connection`?

Reply:
[be11139](https://github.com/interledger/open-payments/pull/128/commits/be11139495288888e5ae7cb761ae0a3185f9b6e4)